### PR TITLE
Fixed issues observed in operator with fluxninja plugin

### PR DESCRIPTION
**Pull request by hdkshingala**
This change is [Reviewable](https://reviewable.io/reviews/fluxninja/aperture/344)

### CODE REVIEW

File: plugins/service/aperture-plugin-fluxninja/pluginconfig/pluginconfig.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ type FluxNinjaPluginConfig struct {
 	APIKey string `json:"api_key,omitempty"`
 	// Client configuration.
 	//+kubebuilder:validation:Optional
+	//+kubebuilder:default:={http:{network_keep_alive:"30s"}}
```
Comment (harjotgill) [inline plugins/service/aperture-plugin-fluxninja/pluginconfig/pluginconfig.go:30]: Why do we have to provide this default over here?

File: pkg/watchdog/policy.go [hunk 2]
```diff
@@ -20,15 +20,19 @@ import (
...
 	//+kubebuilder:validation:Optional
+	//+kubebuilder:default:={watermarks_policy:{watermarks:{0.50,0.75,0.80,0.85,0.90,0.95,0.99}}}
 	CGroup WatchdogPolicyType `json:"cgroup"`
 
 	//+kubebuilder:validation:Optional
+	//+kubebuilder:default:={watermarks_policy:{watermarks:{0.50,0.75,0.80,0.85,0.90,0.95,0.99}}}
 	System WatchdogPolicyType `json:"system"`
 
 	//+kubebuilder:validation:Optional
+	//+kubebuilder:default:={limit:268435456,watermarks_policy:{watermarks:{0.50,0.75,0.80,0.85,0.90,0.95,0.99}}}
```
Comment (harjotgill) [inline pkg/watchdog/policy.go:35]: This is too much code repetion. Are we sure we are using kubebuilder correctly?

Comment (harjotgill) [reply pkg/watchdog/policy.go:35]: we should probably just omitempty in this case

**Comment by harjotgill** [comment]
We can close this.

**Comment by hdkshingala** [comment]
These issues were handled as part of https://github.com/fluxninja/aperture/pull/364.

Closing this PR.

**Pull request closed without merging**